packages/terra-action-footer/themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -0,0 +1,10 @@
+:global {
```

Comment:
@yuderekyu @StephenEsser @tbiethman 
Should these be applied as a global theme? Does this affect other themes by adding these as global? I would have thought these should be aggregated by the theme provider?

Reply:
Posting my reply here too:

We default to global in our webpack config. It doesn't hurt to be explicit. https://github.com/cerner/terra-toolkit/blob/master/config/webpack/webpack.config.js#L65

Yes, these files are aggregated to root or scoped files, which are then included as an entry point:
```
:root {
  @import ...modal-manager/themes...orion-fusion-theme.scss
}
```
```
.orion-fusion-theme {
  @import ...modal-manager/themes...orion-fusion-theme.scss
}
```
Scoping these globally avoids a second scope layer. This should not be an issue because theme files are not imported in the component. Someone could manually import these, but that's on them.